src/assets/src/components/edit.tsx

```diff
@@ -237,9 +237,14 @@ interface MeetingInfoProps {
 const MeetingInfoDialog = (props: MeetingInfoProps) => {
     const generalInfo = props.meeting
         && (
+            <>
             <p>
                 Attendees: {props.meeting.attendees.map(a => <UserDisplay user={a}/>)}
             </p>
+            <p>
+                Agenda: {props.meeting?.agenda}
```

Comment:
I think the safe navigation operator (`?.`) isn't needed here, because `props.meeting` definitely exists. Let me know if I'm mistaken

Reply:
Oops, sorry! This was part of your first comment. Should be fixed now.